dev/mongo-rs-backups/docker-compose.yml

```diff
@@ -0,0 +1,36 @@
+version: "4"
```

Comment:
It's deprecated, see https://docs.docker.com/compose/compose-file/04-version-and-name/